src/services/toggleListForService.ts

```diff
@@ -0,0 +1,13 @@
+import { ToguruToggleData, ToguruData } from '../types/toguru'
+
+const toggleBelongsToService = (toggle: ToguruToggleData, serviceName: string): boolean => {
```

Comment:
switch signature type 

```
```suggestion
const toggleBelongsToService = (serviceName: string) => (toggle: ToguruToggleData): boolean => {
```

then you can write method below nicer